contracts/custom/DecentDAOVoting.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity =0.8.19;
+
+import { LinearERC20Voting } from "../azorius/LinearERC20Voting.sol";
+import { VotesERC20Wrapper } from "../VotesERC20Wrapper.sol";
+
+interface Escrow {
+    function getBalance(address _address, uint _block) external;
```

Comment:
@adamgall to implement what we're currently thinking, the escrow contract would just need to have this function.